DevCycle.SDK.Server.Local.MSTests/EventQueueTest.cs

```diff
@@ -187,5 +218,43 @@ private void AssertFalseFlushedEvents(object sender, DVCEventArgs e)
         {
             Assert.IsFalse(e.Success);
         }
+
+        private bool AssertAggregateEventsBatch(BatchOfUserEventsBatch b)
+        {
+            Debug.Assert(b.UserEventsBatchRecords.Count == 3);
```

Comment:
Any specific reason you're using debug assert instead of just raw Assert?

Reply:
the signature of regular `Assert` seemed different, it wouldn't let me just pass a boolean

Reply:
https://github.com/DevCycleHQ/dotnet-server-sdk/pull/4/files#diff-b57e1224f6a8934aadf5d64ddf0373820157e68693e0856bc2de6531f24727eeR231

This is the signature you'd probably want - Asset has a ton of overloads.